tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+    "compilerOptions": {
```

Comment:
do you want to enable "declaration" to get `.d.ts` files

Reply:
That's still not currently supported.